README.rst

```diff
@@ -26,218 +34,116 @@ Installs the Apache package and starts the service.
+      server_apache_config:
+        directives:
+          - Timeout: 5
+        containers:
+          IfModule:
+            -
+              item: 'mime_module'
+              directives:
+                - AddType: 'application/x-font-ttf ttc ttf'
+                - AddType: 'application/x-font-opentype otf'
+                - AddType: 'application/x-font-woff woff2'
 
-Configures the apache mpm modules on Debian ``mpm_prefork``, ``mpm_worker`` or ``mpm_event`` (Debian Only)
 
 ``apache.modules``
 ------------------
 
 Enables and disables Apache modules.
 
-``apache.mod_rewrite``
-----------------------
-
-Enabled the Apache module mod_rewrite (Debian and FreeBSD only)
-
-``apache.mod_proxy``
--------------------
-
-Enables the Apache module mod_proxy. (Debian and FreeBSD only)
-
-``apache.mod_proxy_http``
--------------------------
-
-Enables the Apache module mod_proxy_http and requires the Apache module mod_proxy to be enabled. (Debian Only)
-
-``apache.mod_proxy_fcgi``
--------------------------
-
-Enables the Apache module mod_proxy_fcgi and requires the Apache module mod_proxy to be enabled. (Debian Only)
-
-``apache.mod_wsgi``
--------------------
-
-Installs the mod_wsgi package and enables the Apache module.
-
-``apache.mod_actions``
-----------------------
-
-Enables the Apache module mod_actions. (Debian Only)
-
-``apache.mod_headers``
-----------------------
-
-Enables the Apache module mod_headers. (Debian Only)
-
-``apache.mod_pagespeed``
-------------------------
-
-Installs and Enables the mod_pagespeed module. (Debian and RedHat Only)
-
-``apache.mod_perl2``
--------------------
-
-Installs and enables the mod_perl2 module (Debian and FreeBSD only)
-
-``apache.mod_geoip``
--------------------
-
-Installs and enables the mod_geoIP (RedHat only)
-
-``apache.mod_php5``
--------------------
-
-Installs and enables the mod_php5 module
-
-``apache.mod_cgi``
----------------------
-
-Enables mod_cgi. (FreeBSD only)
-
-``apache.mod_fcgid``
---------------------
-
-Installs and enables the mod_fcgid module (Debian only)
-
-``apache.mod_fastcgi``
---------------------
-
-Installs and enables the mod_fastcgi module
-
-``apache.mod_dav_svn``
---------------------
-
-Installs and enables the mod_dav_svn module (Debian only)
-
-``apache.mod_security``
-----------------------
-
-Installs an enables the `Apache mod_security2 WAF`<http://modsecurity.org/>`_
-using data from Pillar. (Debian and RedHat Only)
-
-Allows you to install the basic Core Rules (CRS) and some basic configuration for mod_security2
+``apache.vhosts.vhost``
+--------------------------
 
-``apache.mod_security.rules``
------------------------------
+Configures Apache name-based virtual hosts and creates virtual host directories using data from Pillar.
 
-This state can create symlinks based on basic Core Rules package. (Debian only)
-Or it can distribute a mod_security rule file and place it /etc/modsecurity/
+All necessary data must be provided in the pillar
```

Comment:
@aboe76 @javierbertoli I'd merge this PR as-is, but do we want to add another PR which implements the TOFS pattern?

- https://github.com/saltstack-formulas/template-formula/pull/19
- https://github.com/saltstack-formulas/template-formula/issues/21
Ctrl+F "TOFS pattern vs. pillar"

Reply:
@alxwr Let's finalise the portable TOFS implementation in https://github.com/saltstack-formulas/systemd-formula/pull/27.  This should lead to a drop-in solution using `libtofs.jinja`.